Assignments/Sorting/SelectionSort.playground/Pages/SequenceType Extension.xcplaygroundpage/Contents.swift

```diff
@@ -6,11 +6,41 @@ Instead of only sorting arrays, modify your function to sort any type conforming
 */
 
 extension SequenceType {
+  func selectionSort(@noescape isOrderedBefore: (Generator.Element, Generator.Element) -> Bool) -> [Generator.Element] {
+    func findMinIndex(array: [Generator.Element], startingFrom start: Int = 0, @noescape isOrderedBefore: (Generator.Element, Generator.Element) -> Bool) -> Int? {
```

Comment:
Thank you for the PR @matteomatassoni. One suggestion: we can turn this `findMinIndex` to an extension of `Array` maybe. What do you say?